label/label.go

```diff
@@ -62,7 +62,7 @@ func New(repo, pkg, name string) Label {
 var NoLabel = Label{}
 
 var (
-	labelRepoRegexp = regexp.MustCompile(`^@$|^[A-Za-z.-][A-Za-z0-9_.-]*$`)
+	labelRepoRegexp = regexp.MustCompile(`^@$|^[A-Za-z.-][A-Za-z0-9_.~-]*$`)
```

Comment:
I opted for the smallest version of a fix here, looking at master in Bazel: 
https://github.com/bazelbuild/bazel/blob/71fb1e4188b01e582a308cfe4bcbf1c730eded1b/src/main/java/com/google/devtools/build/lib/cmdline/LabelValidator.java#L272-L278

...it seems as if there are no invalid characters, and that a repo name is any string between `@` and `//`. That calls for a bit more rework to this method, and I was reluctant to propose a bigger change as a first time contributor.